### PR TITLE
Use Drupal route pattern for edits

### PR DESCRIPTION
**Pull request by mattgrill**
## Testing instructions

- Make sure `/node/:nid/edit` works

### CODE REVIEW

File: src/utils/api/api.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ async function api(
       options.headers.Accept = 'application/vnd.api+json';
       break;
     case 'content_single':
-      url = `/jsonapi/node/${parameters.bundle}/${parameters.id}`;
```
Comment (dawehner) [inline src/utils/api/api.js]: How about not touching the content_single endpoint but keep using the content one instead, given it is now identical?

File: src/components/05_pages/NodeEditForm/index.js [hunk 2]
```diff
@@ -32,21 +38,18 @@ NodeEditForm.defaultProps = {
...
-  bundle: PropTypes.string.isRequired,
-  id: PropTypes.string.isRequired,
+  nid: PropTypes.string.isRequired,
   requestSingleContent: PropTypes.func.isRequired,
   entity: PropTypes.shape({
-    data: PropTypes.shape({
-      attributes: PropTypes.shape({
-        title: PropTypes.string.isRequired,
-      }).isRequired,
+    attributes: PropTypes.shape({
```
Comment (dawehner) [inline src/components/05_pages/NodeEditForm/index.js]: It feels like a follow up could expand the prop type a bit more to include relationships as well?

File: src/components/05_pages/NodeEditForm/index.js [hunk 3]
```diff
@@ -8,20 +8,26 @@ import PageTitle from '../../02_atoms/PageTitle/PageTitle';
...
 
   render() {
-    const { entity, bundle } = this.props;
+    const { entity } = this.props;
     return (
       <Fragment>
         {entity && (
           <PageTitle>
-            Edit {bundle} {entity.data.attributes.title}
+            Edit {entity.type.replace('node--', '')} {entity.attributes.title}
```
Comment (dawehner) [inline src/components/05_pages/NodeEditForm/index.js]: Should we add the bundle as variable here?

File: src/components/05_pages/NodeEditForm/index.js (hunk 2 above)
Comment (mattgrill) [reply src/components/05_pages/NodeEditForm/index.js]: Yes. I want to create a followup to address PropTypes across all our components.

File: src/utils/api/api.js (hunk 1 above)
Comment (mattgrill) [reply src/utils/api/api.js]: Yes. Good point!

**Pull request merged**